sharepoint/sharepoint-ps/sharepoint-online/Get-SPOApplication.md

```diff
@@ -50,15 +50,16 @@ Example 1 returns all SharePoint Embedded applications registered in the specifi
 Get-SPOApplication -OwningApplicationId <OwningApplicationId>
 ```
 
-Example 2 lists the details of the owning application corresponding to the `OwningApplicationId` registered in the specified tenant.
+Example 2 lists the details of the owning application corresponding to the `OwningApplicationId` registered in the specified tenant. This returns `Applications` which specifies the list of guest applications IDs with current access permissions to the owning application, `SharingCapability` settings, and `OverrideTenantSharingCapability` status.
+
 
 ### Example 3
 
 ```powershell
 Get-SPOApplication -OwningApplicationId <OwningApplicationId> -ApplicationId <ApplicationId>
 ```
 
-Example 3 enumerates permissions of the owning applications registered in the specified tenant.
+Example 3 list details of the guest application specified in `ApplicationId`. This inclues the list of guest application access permissions. Note that only app-only permissions are supported.
```

Comment:
>are supported [](http://example.com/codeflow?start=177&length=13)

it's not clear what this "are supported" is in reference to.

Reply:
line 97 is clearer in that it clarifies this "app-only" refers to the returned items. consider rewording this to something like line 97?